frontends/web/src/components/layout/guide-wrapper.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2023 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ReactNode } from 'react';
+import styles from './guide-wrapper.module.css';
+
+type TProps = {
+  children: ReactNode;
+}
+
+export const GuideWrapper = ({ children }: TProps) => {
+  return (
+    <div className={styles.contentWithGuide}>
+      {children}
+    </div>
+  );
+};
+
+export const GuidedContent = ({ children }: TProps) => {
+  return (
+    <div className={styles.container}>
+      {children}
+    </div>
+  );
+};
```

Comment:
there 2 div's are basically the same as it was before with https://github.com/digitalbitbox/bitbox-wallet-app/commit/484c42f59207f12f428f588935cd3628e603d83b#diff-21e6bfd8c5f3306cafbabf33252393a5fb8e9ddf3d23bd7b42a417d74d038e7dL64-L65

with the expection of the scrollcontainer, which is the reason that in the CSS there is an additional definition `overflow: auto`.